pipelines/deployer/deployer.yaml

```diff
@@ -217,6 +218,14 @@ apply_cluster_chart: &apply_cluster_chart
         --values config/${CONFIG_VALUES_PATH} \
         --set global.runningOnAws=true \
         gsp/gsp-istio-*.tgz
+      kubectl delete -R -f manifests/gsp-isti/istio-cni
```

Comment:
Can we change `gsp-isti` to `gsp-istio`? Or is `gsp-isti` valid directory?

Reply:
Oof. You're right. Good catch.